src/main/groovy/co/freeside/betamax/XmlAwareResponseBodyMatcher.groovy

```diff
@@ -0,0 +1,27 @@
+package co.freeside.betamax
+
+import co.freeside.betamax.message.Response
+
+/**
+ * A rule for matching response bodies
+ * while ignoring specified xml elements.
+ */
+class XmlAwareResponseBodyMatcher implements Comparator<Response> {
+  List<String> xmlElementNames = []
+
+  XmlAwareResponseBodyMatcher(String... xmlElementNames) {
+    this.xmlElementNames = xmlElementNames
+  }
+
+  int compare(Response a, Response b) {
+    (a.hasBody() ? cleanse(a.bodyAsText.text) : null) <=>
+    (b.hasBody() ? cleanse(b.bodyAsText.text) : null)
+  }
+
+  // Does not work with self-nested element, e.g. <a><a>1</a></a>
+  String cleanse(String bodyText) {
```

Comment:
I was going to wait and see if we had the need.  I think if we do we have to use an XML parser - I've read that regexp cannot handle this kind of thing.  I believe it's because regexp does not work for context-free grammars, and I think any grammar that supports arbitrarily deep nesting is technically not a regexp fit.